crates/ty_project/src/db/changes.rs

```diff
@@ -52,7 +52,7 @@ impl ProjectDatabase {
             custom_stdlib_changed: false,
         };
         // Paths that were added
-        let mut added_paths = FxHashSet::default();
+        let mut added_paths = BTreeSet::default();
```

Comment:
Same here. I don't see why a stable ordering would be necessary here. We sort the returned diagnostics